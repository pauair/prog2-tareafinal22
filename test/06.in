agregarNuevoProducto 50 prod50
definirFecha 01/05/2022
agregarUnidades 50 2 01/05/2023
agregarUnidades 50 5 01/05/2024
agregarUnidades 50 2 01/05/2025
agregarUnidades 50 5 01/05/2026
vencimientosProducto 50
definirFecha 01/04/2023
vencimientosProducto 50
definirFecha 01/05/2025
vencimientosProducto 50
definirFecha 01/05/2027
vencimientosProducto 50
Fin