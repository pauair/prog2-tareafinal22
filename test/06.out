1>Producto agregado 50 - prod50.
2>Fecha definida en forma exitosa.
3>Al producto 50 se le agregaron 2 unidades con fecha de vencimiento 1/5/2023.
4>Al producto 50 se le agregaron 5 unidades con fecha de vencimiento 1/5/2024.
5>Al producto 50 se le agregaron 2 unidades con fecha de vencimiento 1/5/2025.
6>Al producto 50 se le agregaron 5 unidades con fecha de vencimiento 1/5/2026.
7>Informaci贸n del producto 50 - prod50
Total de unidades: 14
1/5/2023 -> 2 
1/5/2024 -> 5 
1/5/2025 -> 2 
1/5/2026 -> 5 
8>Fecha definida en forma exitosa.
9>Informaci贸n del producto 50 - prod50
Total de unidades: 14
1/5/2023 -> 2 
1/5/2024 -> 5 
1/5/2025 -> 2 
1/5/2026 -> 5 
10>Fecha definida en forma exitosa.
11>Informaci贸n del producto 50 - prod50
Total de unidades: 5
1/5/2026 -> 5 
12>Fecha definida en forma exitosa.
13>Informaci贸n del producto 50 - prod50
Total de unidades: 0
14>Fin.
